docs/src/content/docs/reference/scripts/system.mdx

```diff
@@ -4275,17 +4275,17 @@ export default function (ctx: ChatGenerationContext) {
 
 ### `system.zero_shot_cot`
 
-Zero-shot Chain Of Though
+Zero-shot Chain Of Thought
```

Comment:
This file is built from system.zcot.genai.mts so we need to fix that first. Will do.

Reply:
so why didn't you reject this PR?

Reply:
the typo in returns is legit and you spent time crafting a PR which pointed to the issue. Worth a merge.